packages/atomic/src/components.d.ts

```diff
@@ -289,6 +289,16 @@ export namespace Components {
     }
     interface AtomicFrequentlyBoughtTogether {
     }
+    interface AtomicHtml {
+        /**
+          * Specify if the content should be sanitized, using `DOMPurify` (https://www.npmjs.com/package/dompurify).
```

Comment:
```suggestion
          * Specify if the content should be sanitized using [`DOMPurify`](https://www.npmjs.com/package/dompurify).
```

For this and other occurrences of this phrase 🙏